Dockerfile

```diff
@@ -22,4 +22,4 @@ COPY --from=bootstrap /work/compiler/stage0/lib/ /usr/lib/
 FROM skiplang AS skip
 
 RUN --mount=type=bind,source=./bin/apt-install.sh,target=/tmp/apt-install.sh \
-    /tmp/apt-install.sh skipruntime-deps other-CI-tools
+    /tmp/apt-install.sh skipruntime-deps other-CI-tools other-dev-tools
```

Comment:
This docker image is used by the CI.

Ah, maybe you use it for development too?

Reply:
Oh, you don't? :-) My understanding was that all the build and whatnot workflow was expecting us to be using these docker images, possibly extended with things for our personal use in `skip/$USER/Dockerfile` where the image build script looks for user-specific stuff.

Reply:
I work on Ubuntu, things just work :-D

We can split up the image further if needed later, let's keep moving